src/pages/basket/listeners/changeQuantity/plusMinusOneProduct.ts

```diff
@@ -0,0 +1,24 @@
+import { decreaseItemInBasket, increaseItemInBasket } from '../../../../server/cart/changeQuantity';
+import { recalculate } from '../../functions/recalculate';
+import { minusOne } from './minusOne';
+import { plusOne } from './plusOne';
+
+export function plusMinusOneProduct(): void {
+    const count = 1;
+    document.addEventListener('click', async (event) => {
+        const target = event.target as HTMLButtonElement;
+        if (target.className === 'plus-button') {
+            const id = target.id.replace('-cart-prod-plus', '');
+            plusOne(target, count);
+            const cart = await increaseItemInBasket(id);
+
+            recalculate(id, cart?.totalPrice.centAmount);
+        } else if (target.className === 'minus-button') {
+            const id = target.id.replace('-cart-prod-minus', '');
```

Comment:
i see that you just need to make id empty, right? if yes just remove it from 11 and 17 lines

Reply:
No, i need to separate id of product from button id

Not corrected